news/_posts/2024-06-06-dada2-tutorial.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: news
+title: "New Galaxy training: Building an amplicon sequence variant (ASV) table from 16S data using DADA2"
+contributions:
+  authorship: 
+  - bebatut
+tags: 
+- new tutorial
+tutorial: topics/microbiome/tutorials/dada-16S/tutorial.html
+---
+
+The GTN hosts its [400th tutorial]({% link news/_posts/2024-06-06-400-tutorials-milestone.md %}): **Building an amplicon sequence variant (ASV) table from 16S data using DADA2**
```

Comment:
@hexylena any idea why the linting is failing on this link

Reply:
yes :sweat: we have a custom link plugin to work around a jekyll bug (which we might be able to revert now) but  it never supported news posts

```suggestion
The GTN hosts its [400th tutorial]({{ site.baseurl }}/news/2024/06/06/400-tutorials-milestone.html): **Building an amplicon sequence variant (ASV) table from 16S data using DADA2**
```